src/attack.rs

```diff
@@ -40,6 +41,19 @@ pub struct Attack {
     pub velocity: Vec2,
 }
 
+/// A component that depawns an entity after collision with tolerance to x collisions.
+#[derive(Component, Clone, Copy, Default, Reflect)]
+pub struct Breakable {
+    pub hits: i32,
+    count: i32,
```

Comment:
My gut reaction is that this should be public. Seems like we might as well allow other systems to modify it it sees fit ( maybe mods for example ). It doesn't really hurt anything other than maybe producing unexpected behavior, which if somebody is modifying that field manually, then it's kind of already obvious what they're messing with.

I you feel like we should keep it private, though, I'm not super set on making it public.

If we do make it public, maybe name the fields, `hit_tolerance` and `hit_count`, and add super minimal doc strings on them to help differentiate what they are for?